charon/tests/cargo/non-host-target/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+#![no_std]
+
+pub fn add(left: u64, right: u64) -> u64 {
```

Comment:
Could you add some target-dependent code here to we're sure that we're actually using the correct target?

Reply:
Actually, I would like this package to be able to compile on both host and non-host target. So it shouldn't be target-dependent.

The only requirement I think is cross-compilation is feasible on host machine.

Reply:
I'll add a comment for this package tho.

Or if you want this package to be only compile on single non-host target?

Reply:
I meant something like a `cfg(target = ...)` so we can observe a difference in the compiled code